### ISSUE TITLE
Config is missing MacAddress

### ISSUE THREAD

**Issue by paulbellamy** [fsouza/go-dockerclient#283]
When creating a container, you can also specify the MacAddress, therefore Config should include:
```Go
type Config struct {
  ...
  MacAddress string `json:"MacAddress,omitempty" yaml:"MacAddress,omitempty"`
  ...
}
```

**Comment by fsouza** [comment]
Fixed via #284.

**Comment by paulbellamy** [comment]
Excellent! Thanks so much. :)
